plugin/pkg/admission/resourcequota/config.go

```diff
@@ -37,19 +37,35 @@ func init() {
 	install.Install(scheme)
 }
 
+func defaultLimitedResources() []resourcequotaapi.LimitedResource {
+	return []resourcequotaapi.LimitedResource{
+		{
+			Resource: "pods",
+			MatchScopes: []corev1.ScopedResourceSelectorRequirement{
+				{
+					ScopeName: corev1.ResourceQuotaScopePriorityClass,
+					Operator:  corev1.ScopeSelectorOpIn,
+					Values:    []string{"system-cluster-critical", "system-node-critical"},
+				},
+			},
+		},
+	}
+}
+
+func DefaultAdmissionConfig() *resourcequotaapi.Configuration {
+	return &resourcequotaapi.Configuration{
+		LimitedResources: defaultLimitedResources(),
+	}
+}
+
 // LoadConfiguration loads the provided configuration.
 func LoadConfiguration(config io.Reader) (*resourcequotaapi.Configuration, error) {
 	// if no config is provided, return a default configuration
 	if config == nil {
-		externalConfig := &resourcequotav1beta1.Configuration{}
-		scheme.Default(externalConfig)
```

Comment:
did you just lose defaulting?

Reply:
I did that purposefully. I did not notice any defaulterFuncs for this type. Not sure, if this is needed for every type? Do we need to?

Reply:
Added it back.